### DISCUSSION TITLE
glass material in sceneviewer

### DISCUSSION THREAD

**Discussion by devhims** [google/model-viewer#3638] [Q&A]
I'm using a texture with alpha blend for the glass material and it works well in 3d view, webxr, and quick look modes. However, I get a weird shade with scene viewer on Android (like in 2nd screenshot). Is there is a recommended fix for this issue? 

<img width="500" alt="Screenshot 2021-03-20 at 2 44 47 AM" src="https://user-images.githubusercontent.com/17825870/111842960-4ab3ec00-8926-11eb-9029-ef8a9b48e0fb.png">

<img width="500" alt="Screenshot 2021-03-20 at 2 44 06 AM" src="https://user-images.githubusercontent.com/17825870/111842952-48519200-8926-11eb-919d-a54aeb356e33.png">

**Comment by elalish** [comment]
@tpsiaki for the Scene Viewer bug.

**Comment by elalish** [comment]
@devhims Can you please put a link to your repro URL so we can inspect your GLB? It'll be hard to repro otherwise.

**Comment by devhims** [comment]
@elalish @tpsiaki you can download the GLB on this link:
https://cdn.glitch.com/5a4e620e-8d0d-4d82-a44f-08f272f0dd9d%2FBenzGLB3.glb?v=1616183257719

**Comment by sercandemirtas** [comment]
Hey @devhims

I download your model and checked in blender it looks like there is no problem on shading.

This is demo link for your model
https://relaxed-neumann-b6600b.netlify.app/orjinal.html

and opened in scene viewer Android as you see in the picture below the result is good. Which device did you tried your project? Maybe something is wrong on your codes.

![WhatsApp Image 2021-03-29 at 23 35 41](https://user-images.githubusercontent.com/68244546/112897183-f5c36300-90e7-11eb-96c3-179cd978688e.jpeg)

**Comment by devhims** [comment]
Hi @sercandemirtas, I tested the link. I'm pretty sure the mode is webxr and not scene viewer. On Android with model viewer, that's the default mode unless you change it.
